src/featureflagservice/priv/repo/migrations/20220524172636_create_featureflags.exs

```diff
@@ -25,6 +25,11 @@ defmodule Featureflagservice.Repo.Migrations.CreateFeatureflags do
       name: "recommendationCache",
       description: "Cache recommendations",
       enabled: false})
+
+    repo().insert(%Featureflagservice.FeatureFlags.FeatureFlag{
+      name: "adServiceFailure",
+      description: "Fail ad service requests sporadically",
+      enabled: false})
```

Comment:
Copy and pasted. Do I need to do anything else to ensure that migration runs and applies this change?

Reply:
Great question,  I think @tsloughter would be able to help us out.
If I'm not mistaken @puckpuck also added a FeatureFlag previously.

Reply:
@jack-berg, see the last 5 lines of this file. I think the new flag should be added also there.

Reply:
Thanks!